habitat-baselines/habitat_baselines/rl/hrl/hierarchical_policy.py

```diff
@@ -118,6 +115,13 @@ def __init__(
     def eval(self):
         pass
 
+    def get_policy_action_space(
+        self, env_action_space: spaces.Space
+    ) -> spaces.Space:
```

Comment:
Add a comment here. I am not sure if this returns the HL or LL action space looking at this code.